packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.html

```diff
@@ -39,8 +41,11 @@
             </div>
             <div class="nui-widget__header__content-actions">
                 <a *ngIf="url && !editMode" nui-button
+                   #link
                    displayStyle="action"
                    [href]="url"
+                   (click)="changeLink($event)"
+                   (mouseenter)="changeLink($event)"
```

Comment:
again, maybe the call on mouseenter is enough? it would be hard to click it without mouseentering it first